azurerm/internal/services/web/app_service.go

```diff
@@ -490,6 +491,19 @@ func schemaAppServiceSiteConfig() *pluginsdk.Schema {
 					Type:     pluginsdk.TypeString,
 					Optional: true,
 				},
+
+				"acr_use_managed_identity_creds": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
```

Comment:
We should probably make this computed only and use the presence of `acr_user_managed_identity_id` to toggle this on / off? We also try to favour long-hand names for clarity.
```suggestion
				"acr_use_managed_identity_credentials": {
					Type:     pluginsdk.TypeBool,
					Computed: true,
				},
```

Reply:
this would not allow the use of the app service system assigned identity only user managed identities. So I will not be making that change as that is not the use case I need.

Reply:
My mistake - I'm in the process of rewriting this service in preparation for 3.0 and my notes on this feature were incorrect. I'll dismiss the review and re-do it asap.